### ISSUE TITLE
Scrypted apt repository not signed

### ISSUE THREAD

**Issue by mandreko** [koush/scrypted#1482]
**Describe the bug**
The apt repository for Scrypted is not GPG signed. Because of this, installation of the desktop electron client on Ubuntu 24+, will fail, due to Ubuntu's new signing requirements.

**To Reproduce**
1. Following the installation guide, run:
```
echo 'deb [trusted=yes] https://nuts.scrypted.app/apt stable main' | sudo tee /etc/apt/sources.list.d/scrypted.list
sudo apt update
```

**Expected behavior**
The apt update should complete successfully, creating the index of installable applications.

**Screenshots**
![image](https://github.com/koush/scrypted/assets/419731/e87cf9ef-9ac4-4f10-8f37-76f5698096a7)

**Server (please complete the following information):**
N/A

**Hardware Model (please complete the following information):**
N/A

**Client (please complete the following information, if applicable):**
 - OS: Ubuntu 24.04

**Additional context**
I've started looking at Ubuntu's new apt requirements. I'm trying to see if there's a way to force a trust, but haven't identified one yet.

**Comment by koush** [comment]
I'll get that fixed. I recommend running Scrypted in docker/proxmox regardless, this app is intended only to be used as a viewer.

**Comment by mandreko** [comment]
Yep, I only use this on my desktop as a viewer to my NVR running on a proxmox server. ;)

Thanks a bunch! I *think* it should be a fairly easy to fix, but I've heard managers tell me that before when I was a software developer too...

**Comment by crissmil** [comment]
latest release 0.109 is not available on homeassistant add-on. Still on 0.105 and cannot update

**Comment by koush** [comment]
turning out to be a bit of a pain in the ass to handle.
